examples/speaker_tasks/diarization/conf/offline_diarization_with_asr.yaml

```diff
@@ -33,6 +33,7 @@ diarizer:
     parameters:
       window_length_in_sec: 1.5 # window length in sec for speaker embedding extraction
```

Comment:
The above comment applies to this yaml file as well.

Reply:
This will be updated too.